lib/pysquared/pysquared.py

```diff
@@ -434,6 +434,13 @@ def __init__(self) -> None:
         """
         self.scan_tca_channels()
 
+        """
+        Camera Initializations
+        """
```

Comment:
```suggestion
```
I'm not sure why this was in the "Camera Initializations" section but this code modifies the radio so let's at least remove the comments.